website/src/remark/render-code-blocks.mjs

```diff
@@ -167,16 +167,23 @@ export default function plugin() {
         visit(ast, 'code', (node, index, parent) => {
             const { value, lang, meta } = node;
 
-            const skipRegex = new RegExp(/(no-build)|(ignore-errors)/g);
-            const skipErr = meta && meta.match(skipRegex) !== null;
-            const editableRegex = new RegExp(/(always-editable)/g);
-            const alwaysEditable = meta && meta.match(editableRegex) !== null;
+            const skipRegex = /(no-build)|(ignore-errors)/;
+            const skipErr = skipRegex.test(meta);
+            const editableRegex = /(always-editable)/;
+            const alwaysEditable = editableRegex.test(meta);
+            const lineNumRegex = /(showLineNumbers)/i;
```

Comment:
Pixies camel case and dashed naming schemes... pick one :)

Reply:
dashed naming it is!